lib/standard/loads_yaml_config.rb

```diff
@@ -12,11 +12,25 @@ def initialize
     def call(argv, search_path)
       yaml_path = @parses_cli_option.call(argv, "--config") ||
         FileFinder.new.call(".standard.yml", search_path)
-      construct_config(yaml_path, load_standard_yaml(yaml_path))
+      standard_yaml = load_standard_yaml(yaml_path)
+
+      todo_yaml_path = @parses_cli_option.call(argv, "--todo") ||
+        FileFinder.new.call(".standard_todo.yml", search_path)
+      todo_yaml = load_standard_yaml(todo_yaml_path)
+
+      merge_ignore(standard_yaml, todo_yaml)
+      construct_config(yaml_path, standard_yaml)
```

Comment:
I'd at least reorder the body to put the two yaml files next to each other, but better yet since they do the same thing, extract that to a method, so that the duplication is gone and all 3 things the method does are at the same level of abstraction

Reply:
Refactored to what I think you are asking for.  Let me know if I misunderstood.